$NetBSD: patch-aa,v 1.1 2009/01/22 08:50:12 obache Exp $

Fix: redefinition of macro SP (workarround).
http://git.sourceforge.jp/view?p=nkf/nkf.git;a=commitdiff;h=d2f5d12b00a610a371ec48f6c841953b792aa721

NKF.xs must follow nkf.c doesn't have WISH_TRUE AND NO_X0201
http://git.sourceforge.jp/view?p=nkf/nkf.git;a=commitdiff;h=f06661980bb179714f60338e09d48ae66ca15f97

--- NKF.mod/NKF.xs.orig	2009-01-20 09:49:31.000000000 +0000
+++ NKF.mod/NKF.xs
@@ -95,7 +95,9 @@ nkf_putchar_grow(unsigned int c) 
 
 #define PERL_XS 1
 #include "../utf8tbl.c"
+#undef SP
 #include "../nkf.c"
+#define SP sp /* perl's CORE/pp.h */
 
 /* package defenition  */
 
@@ -134,9 +136,6 @@ nkf(...)
     data = SvPV(ST(argc),i_len);
     input_ctr = 0;
 
-    if(x0201_f == WISH_TRUE)
-         x0201_f = ((!iso2022jp_f)? TRUE : NO_X0201);
-
     /* allocate the result buffer */
 
     /* During conversion, stirngs length may grow. This is the unit */
@@ -178,9 +177,6 @@ nkf_continue(...)
     data = SvPV(ST(0),i_len);
     input_ctr = 0;
 
-    if(x0201_f == WISH_TRUE)
-         x0201_f = ((!iso2022jp_f)? TRUE : NO_X0201);
-
     /* allocate the result buffer */
 
     /* During conversion, stirngs length may grow. This is the unit */
